tests/integration/src/test/java/org/apache/pulsar/tests/integration/golang/GolangSchemaTest.java

```diff
@@ -0,0 +1,113 @@
+/**
```

Comment:
is `golang` a preferable name? if not, shall we name the directory as 'go' and change the file name to `GoSchemaTest`?

Reply:
yes, will fix it